### DISCUSSION TITLE
How can i view my installed snaps?

### DISCUSSION THREAD

**Discussion by way2ex** [MetaMask/snaps#1086] [Q&A] [answered]
As documentation metioned, I can see my installed snaps via the MetaMask setting page:
```
Any snap will need to represent itself and what it does to the end user. Via the MetaMask settings page, the user can see their installed snaps. 
```

But i can not find it. Is in detail page: `chrome://extensions/?id=ljfoeinjpaedjfecbmggjgodbgkmjkjk` ?

**Comment by ritave** [comment, marked as answer]
Hello @zhuqingguang, if you installed your snap correctly, it'll show up in the `Settings->Snaps` page of the extension.

1. Have you confirmed that you're using the Flask release of MetaMask? The icon should be purple instead of orange.
2. Have you confirmed that your snap has installed correctly? Your dApp should receive a success result from `ethereum.request` call.

**Reply by way2ex** [reply]
Thank you very much!
I just can't find Setting. Now I see it when i click the account icon.
